source/agora/consensus/EnrollmentManager.d

```diff
@@ -423,6 +423,35 @@ public class EnrollmentManager
         return false;
     }
 
+    /***************************************************************************
+
+        Get the unregistered enrollments in the block
```

Comment:
The name should reflect that it's the unregistered enrollments, not just enrollments.

Reply:
I changed the function name `getUnregisteredEnrollments`